DotNet/ServiceTests/IntegrationTests/DataAcquisition/FhirApiServiceTests.cs

```diff
@@ -156,6 +156,10 @@ public async Task ExecuteRead_OperationOutcomeIsNoted()
             PatientId = "the-patient",
             ResourceId = "the-patient"
         };
+        var fhirQuery = new FhirQuery
```

Comment:
Would be good to add more unit tests for this, but not necessary for 0.3.0. Consider creating follow-up Task issues for 0.3.0 to add unit tests for things like:
1. simple unit test specifically for `isReference = true`
2. mimic DA logs that would result in many IDs in the query param, and ensure they are batched correctly